plugins/modules/podman_image.py

```diff
@@ -491,15 +492,99 @@ def _get_id_from_output(self, lines, startswith=None, contains=None, split_on='
 
         return layer_ids[-1]
 
+    def _find_containerfile_from_context(self):
+        """
+        Find a Containerfile/Dockerfile path inside a podman build context.
+        Return an empty string if none exist.
+        """
+
+        containerfile_path = ""
+        for filename in [os.path.join(self.path, fname) for fname in ["Containerfile", "Dockerfile"]]:
+            if os.path.exists(filename):
+                containerfile_path = filename
+                break
+        return containerfile_path
+
+    def _get_containerfile_contents(self):
+        """
+        Get the path to the Containerfile for an invocation
+        of the module, and return its contents.
+
+        See if either `file` or `container_file` in build args are populated,
+        fetch their contents if so. If not, return the contents of the Containerfile
+        or Dockerfile from inside the build context.
+        """
+
+        build_file_arg = self.build.get('file') if self.build else None
+        containerfile_contents = self.build.get('container_file') if self.build else None
+
+        container_filename = ""
```

Comment:
```suggestion
        container_filename = None
```

Reply:
Fixed in https://github.com/containers/ansible-podman-collections/pull/811/commits/ec02a05e52ce73c30c485bd8b726d5aaab0cbe49